.github/ISSUE_TEMPLATE/asking-for-help.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Asking for help
+about: You have a question on how to use one of gtk3-rs crates
```

Comment:
That should be gtk4-rs

Reply:
/me would appreciate if there's a slight review before instant merging things :-)

Reply:
Sorry, opening a non-instant merging PR to fix it. :-°